verifier/truststore/truststore.go

```diff
@@ -64,49 +64,49 @@ type x509TrustStore struct {
 // GetCertificates returns certificates under storeType/namedStore
 func (trustStore *x509TrustStore) GetCertificates(ctx context.Context, storeType Type, namedStore string) ([]*x509.Certificate, error) {
 	if !isValidStoreType(storeType) {
-		return nil, fmt.Errorf("unsupported store type: %s", storeType)
+		return nil, TrustStoreError{Msg: fmt.Sprintf("unsupported trust store type: %s", storeType)}
 	}
 	if !file.IsValidFileName(namedStore) {
-		return nil, errors.New("named store name needs to follow [a-zA-Z0-9_.-]+ format")
+		return nil, TrustStoreError{Msg: fmt.Sprintf("named store name needs to follow [a-zA-Z0-9_.-]+ format: %s is invalid", namedStore)}
 	}
 	path, err := trustStore.trustStorefs.SysPath(dir.X509TrustStoreDir(string(storeType), namedStore))
 	if err != nil {
-		return nil, err
+		return nil, TrustStoreError{InnerError: fmt.Errorf("failed to get path of trust store %s with type %s: %w", namedStore, storeType, err)}
```

Comment:
This error message is not actionable or helpful to user. should we just say:
```suggestion
		return nil, TrustStoreError{InnerError: err, Msg: fmt.Sprintf("Something went wrong while trying to read trust store %q of type %q", namedStore, storeType)}
```

Reply:
@priteshbandi is this too broad by saying "Something wrong"? Users may get confused without any reason, ideally, for an error message we should have what happened, what is the reason and suggested actions. In this case, do you mean, the reason is not clear, or reason is clear, but we don't have suggested actions?

Reply:
@priteshbandi Note the detailed error is wrapped in the error. Therefore, the real reason of why failed to get path of trust store is shown to the user.
In addition, due to the error type `TrustStoreError`, caller such as Notation CLI is able to handle this type and show actionable instructions to the user, for example, `Use command 'notation cert add' to create and add trusted certificates to the trust store.` In other words, different callers might have different actionable step based on notation-go's error message.